confmap/provider/secretsmanagerprovider/README.md

```diff
@@ -0,0 +1,11 @@
+## Summary
```

Comment:
can you add a metadata.yaml file? Then add a generated code block to this README.md

Reply:
```
status:
  codeowners:
    active: [driverpt]
```

Does this look good ?

Reply:
yes this helps